chain_info/Moonbase_Relay.yaml

```diff
@@ -0,0 +1,42 @@
+name: Moonbase Relay
+network_type: testnet
+chain:
+    type: relaychain
+    # parent: n/a
+    consensus: Aura
+    sudo: true
+chainspec:
+    http_url: https://github.com/PureStake/moonbeam-networks/blob/master/networks/alphanet/westend-raw-specs.json
+# parent: n/a
+goals:
+    - Relay chain for Moonbase Alpha and other parachains who want to tests in interactions with moonbase.
+repository: https://github.com/paritytech/polkadot
```

Comment:
That one is probably not correct

Reply:
We use the polkadot repo, sometimes our fork, but in practice we do not change anything.